### PR TITLE
Fix blast2tsv module

### PR DESCRIPTION
**Pull request by marieBvr**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

Hi,
I have made a new start with this PR. I hope everything goes well.
I apologise for the inconvenience.

### CODE REVIEW

File: tools/virAnnot/macros.xml
Comment (bgruening) [inline tools/virAnnot/macros.xml:1]: I assume those changes do not belong to this PR

File: tools/virAnnot/macros.xml [hunk 1]
```diff
@@ -1,9 +1,4 @@
 <macros>
-    <xml name="xrefs">
-        <xrefs>
-            <xref type="bio.tools">virannot</xref>
```
Comment (bgruening) [inline tools/virAnnot/macros.xml:4]: why are you removing that?

File: tools/virAnnot/blast2tsv.py [hunk 2]
```diff
@@ -102,6 +110,11 @@ def _read_xml(options):
                 hsp["taxonomy"] = ""
                 hsp["organism"] = ""
                 log.warning("RuntimeError - Taxid not found for " + hsp["accession"])
+            except Exception as err:
+                hsp["tax_id"] = ""
+                hsp["taxonomy"] = ""
+                hsp["organism"] = ""
+                log.warning("Taxid not found for " + hsp["accession"] + ". The error is " + err)
```
Comment (bgruening) [inline tools/virAnnot/blast2tsv.py:117]: You could use here f-strings.

**Comment by marieBvr** [comment]
Hi, 
it seems a dependency I am using is broken (ete3 https://groups.google.com/g/etetoolkit/c/e4YLpPlQ-F8).
For this to work, I need to set up an installation via `git install "git+https://github.com/etetoolkit/ete.git@ete3"`.
How can I do that in the macro?

**Comment by bgruening** [comment]
We would need to fix the conda package.

File: tools/virAnnot/virAnnot_blast2tsv.xml [hunk 3]
```diff
@@ -1,9 +1,8 @@
-<tool id="virannot_blast2tsv" name="virAnnot Blast2tsv" version="1.0.0+galaxy0" profile="21.05">
+<tool id="virannot_blast2tsv" name="virAnnot Blast2tsv" version="1.0.1+galaxy0" profile="21.05">
     <description>convert XML blast results to tabular file with taxonomic informations</description>
     <macros>
         <import>macros.xml</import>
     </macros>
-    <expand macro="xrefs"/>
```
Comment (bgruening) [inline tools/virAnnot/virAnnot_blast2tsv.xml:6]: why are those removed? Do we need to fix something elsewhere?

File: tools/virAnnot/virAnnot_rps2tsv.xml [hunk 4]
```diff
@@ -28,19 +27,19 @@
...
             </output>
          </test>
          <test>
             <param name="rps_xml" value="rps_test.xml"/>
             <param name="max_evalue" value="0.0001"/>
-            <output name="output" file="rps_test.tab" compare="diff" lines_diff="12">
+            <output name="output" file="rps_test.tab">
                 <assert_contents>
-                    <has_text text="pfam01333, Apocytochr_F_C, Apocytochrome F" />
+                    <has_text text="pfam00680, RdRP_1" />
```
Comment (bgruening) [inline tools/virAnnot/virAnnot_rps2tsv.xml:42]: is that really enough as text? Maybe you can use `has_n_lines` https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test or `has_n_columns` here as well?

File: tools/virAnnot/virAnnot_blast2tsv.xml [hunk 5]
```diff
@@ -74,12 +73,13 @@
...
                 <assert_contents>
-                    <has_text text="402" />
+                    <has_text text="pfam13603" />
+                    <has_n_lines n="13" />
                 </assert_contents>
             </output>
             <output name="out_annotation" file="blast2tsv_reads_with_rn.txt">
                 <assert_contents>
-                    <has_text text="8" />
+                    <has_text text="Desulfovibrio" />
```
Comment (bgruening) [inline tools/virAnnot/virAnnot_blast2tsv.xml:82]: more asserts? has_n_lines maybe?

Comment (marieBvr) [reply tools/virAnnot/virAnnot_blast2tsv.xml:73]: I combined a few tests to make it more consistent.

**Comment by bgruening** [comment]
I guess it would be good to keep the version also in a macro. I just bumped the 2 missing version for you.
